pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -18,24 +18,48 @@ package tektonresult
 
 import (
 	"context"
+	"os"
+	"path/filepath"
 
 	mf "github.com/manifestival/manifestival"
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
-	"github.com/tektoncd/operator/pkg/client/clientset/versioned"
 	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
 	"github.com/tektoncd/operator/pkg/reconciler/common"
+	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektoninstallerset/client"
 	occommon "github.com/tektoncd/operator/pkg/reconciler/openshift/common"
+	"knative.dev/pkg/logging"
+)
+
+const (
+	// manifests console plugin yaml directory location
+	internalDBYamlDirectory = "static/tekton-results/internal-db"
 )
 
 func OpenShiftExtension(ctx context.Context) common.Extension {
+	logger := logging.FromContext(ctx)
+
+	version := os.Getenv(v1alpha1.VersionEnvKey)
+	if version == "" {
+		logger.Fatal("Failed to find version from env")
+	}
+
+	internalDBManifest, err := getDBManifest()
+	if err != nil {
+		logger.Fatal("Failed to fetch internal db static manifest")
+
+	}
+
 	ext := openshiftExtension{
-		operatorClientSet: operatorclient.Get(ctx),
+		installerSetClient: client.NewInstallerSetClient(operatorclient.Get(ctx).OperatorV1alpha1().TektonInstallerSets(),
+			version, "results-ext", v1alpha1.KindTektonResult, nil),
```

Comment:
What is `results-ext` ?
I'm sure this is not the `componentVersion`
It should be results version I guess

Reply:
Correct but we don't specify the component version in the extension in other components. `componentName-ext`.  Probably because this is OpenShift specific not upstream component specific.